supersuit/vector/multiproc_vec.py

```diff
@@ -183,6 +183,7 @@ def reset(self, seed=None, options=None):
 
```

Comment:
Don't know much about how this wrapper works but it seems to pass the tests despite only returning the observation, so maybe it's fine. Didn't try adding info but could in the future if we want everything to be consistent (this was the only case left of not returning info)

Reply:
Looking back at previous code in the git blame there doesn't ever seem to be an info return oddly enough